stacks/apps/augury.yml

```diff
@@ -81,6 +81,9 @@ Resources:
       SlowJobErrorsCountMetricName: !Sub SlowJobErrorsCountNewCluster${EnvironmentType}
       TargetingRecordCacheSizeMetricsName: !Sub TargetingRecordCacheSize${EnvironmentType}
       AdFilesS3BucketName: !Select [5, !Split [":", !Ref AdFilesS3BucketArn]]
+      WorkerLoggedErrorsMetricName: !Sub WorkerLoggedErrors${EnvironmentType}
+      SlowWorkerLoggedErrorsMetricName: !Sub SlowWorkerLoggedErrors${EnvironmentType}
+      WebLoggedErrorsMetricName: !Sub WebLoggedErrors${EnvironmentType}
```

Comment:
I believe CloudFormation now supports dimensions on metric filters. This should allow us to have a single metric name (e.g., `WebLoggedErrors`), and something like an `Environment` dimension whose value is `${EnvironmentType}`, getting us away from having to make these kind of gross metric names that cram the env type in there, and no longer need to create parameters for the names.

I haven't had a chance to try it, so there's a chance the docs have been updated but the functionality isn't actually supported yet, but this seems like a good chance to test.

The alarms would also have to be updated to add a dimension.

Reply:
I went down the road a bit and hit a stumbling block with the metric filter dimensions.  It seems that the dimensions are derived from the log line itself.  [The `Dimension` property on the metric filter](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-logs-metricfilter-dimension.html) has this caveat:

```
This dimension will only be published for a metric if the value is found in the log event.
```

Which makes me think that a "constant" value would somehow try to match the whole log line, or something else unexpected.  Still worth pursuing?

Reply:
That's annoying. There may be some clever way to inject those values using the filter expression, but I'd have to mess around with it to see. Let's punt on this for now.